lib/ansible/modules/windows/win_wait_for_process.py

```diff
@@ -66,34 +89,43 @@
 EXAMPLES = r'''
 - name: Wait 300 seconds for all Oracle VirtualBox processes to stop. (VBoxHeadless, VirtualBox, VBoxSVC)
   win_wait_for_process:
-    process_name: "v(irtual)?box(headless|svc)?"
+    process_name: 'v(irtual)?box(headless|svc)?'
     state: absent
     timeout: 500
 
-
 - name: Wait 300 seconds for 3 instances of cmd to start, waiting 5 seconds between each check
   win_wait_for_process:
-    process_name: "cmd\\.exe"
+    process_name_exact: cmd
     state: present
     timeout: 500
     sleep: 5
     process_min_count: 3
-
 '''
 
 RETURN = r'''
 elapsed:
-  description: The elapsed seconds between the start of poll and the end of the
-    module.
+  description: The elapsed seconds between the start of poll and the end of the module.
   returned: always
   type: float
   sample: 3.14159265
-changed:
-  description: True if a process was started or stopped during the module execution.
-  returned: always
-  type: bool
 matched_processes:
-  description: Count of processes stopped or started.
+  description: List of matched processes (either stopped or started)
   returned: always
-  type: int
+  type: complex
+  contains:
+    Id:
```

Comment:
I don't like the fact these aren't snake_cake, surely we can convert

Reply:
Hints to do this efficiently in Powershell are welcome :)

Reply:
In this case I would just manually do it, especially turning `Id` to `pid` to match the option names. Otherwise there is https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/powershell/Ansible.ModuleUtils.CamelConversion.psm1.

Reply:
I fixed this. Please re-review :-)